README.md

```diff
@@ -54,13 +54,20 @@ $circuitBreaker = new MainCircuitBreaker(
     $dispatcher
 );
 
+use Nyholm\Psr7\Factory\Psr17Factory;
+
 /**
- * @var Service $service
+ * @var Request $request the PSR-7 request
```

Comment:
MIssing `@return` statement

Reply:
What about having the unsuccessful response object as an argument as well?
We discussed this a while back, this would allow to have more intelligent fallback which could change their fallback return based on the failure (`not found` != `access denied` for example) Unless the request object allows to have this data but that would surprise me

Reply:
I agree, it's hidden but it's already possible in our libraries to retrieve the exception from the fallback. If we make the access of the exception clear enough, do we need to retrieve the Response ?

I'm evaluating the pros/cons of build/maintain/test such object :)